.github/workflows/brim-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Brim release
+
+on:
+  push:
+    tags:
+      - v*brim*
+
+jobs:
+  macos:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - run: git submodule update --init --recursive
+      - run: brew install bison openssl
+      # Prefer static libraries.  Appending to this file is hacky but effective.
+      - run: echo 'set(CMAKE_FIND_LIBRARY_SUFFIXES .a ${CMAKE_FIND_LIBRARY_SUFFIXES})' >> cmake/CommonCMakeConfig.cmake
```

Comment:
I did this to keep CMakeList.txt out of this change, but if it strikes anyone as too dicey, we could instead add the `set(...)` to that file.

Reply:
We can do this after I move henridf/cmake to brimsec/cmake (and try to contribute it to upstream zeek and cmake, possibly enabled by a configure flag?).

Reply:
Would you please add the changes to the CMakeList.txt, along with a comment like: `# Brim: Prefer static ...`.

Reply:
👍